DIRECTORY.md

```diff
@@ -1,130 +1,137 @@
 
 ## Ciphers
   * Caesar
-    * [Caesar Cipher](https://github.com/TheAlgorithms/Go/blob/master/ciphers/caesar/caesar_cipher.go)
+    * [Caesar](https://github.com/TheAlgorithms/Go/blob/master/ciphers/caesar/caesar.go)
     * [Caesar Test](https://github.com/TheAlgorithms/Go/blob/master/ciphers/caesar/caesar_test.go)
   * Diffie Hellman Key Exchange
-    * [Diffie Hellman Key Exchange](https://github.com/TheAlgorithms/Go/blob/master/ciphers/diffie_hellman_key_exchange/diffie_hellman_key_exchange.go)
-    * [Diffie Hellman Key Exchange Test](https://github.com/TheAlgorithms/Go/blob/master/ciphers/diffie_hellman_key_exchange/diffie_hellman_key_exchange_test.go)
+    * [Diffiehellmankeyexchange](https://github.com/TheAlgorithms/Go/blob/master/ciphers/diffie_hellman_key_exchange/diffiehellmankeyexchange.go)
```

Comment:
Too bad, we need this for the algorithms website

Reply:
Can the algorithms website be modified to follow the Go language naming conventions?

Reply:
I guess it will be created automatically, and every name changed. It's not good at all

Reply:
@cclauss the website just parses this file. The problem is that we cannot build it automatically from module/directory names because there are no separators between words

Reply:
@siriak  The website doesn't use the `directory.md` file anymore

@cclauss That shouldn't be necessary, the file names in this pull request should still work with the website.

Reply:
Lets move the list of algorithms to the README and delete DIRECTORY and this workflow then

Reply:
Not all users go to the website.  Many only visit the repo.

Reply:
It will, just in README.md, not DIRECTORY.md

Reply:
And it'll be visible once one opens the main repository page